charts/warpstream-agent/Chart.yaml

```diff
@@ -3,7 +3,7 @@ name: warpstream-agent
 description: WarpStream Agent for Kubernetes.
 type: application
 version: 0.15.89
-appVersion: v730
+appVersion: v731
```

Comment:
oh sorry should have been more clear, it's `version` that has to be bumped, not `appVersion`